pkg/storkctl/action.go

```diff
@@ -41,9 +44,11 @@ func newFailoverCommand(cmdFactory Factory, ioStreams genericclioptions.IOStream
 				return
 			}
 			for _, namespace := range namespaces {
-				if anyActionIncomplete(namespace) {
+				if incompleteAction := getAnyIncompleteAction(namespace); incompleteAction != nil {
```

Comment:
On a second thought I think for all the provided namespace we should check if there is any pending action and error out instead of starting the action for a subset of namespaces. In this way the user gets a chance to fix their list of namespaces and trigger an action in one shot.